app/assets/stylesheets/calagator/index.scss

```diff
@@ -5,6 +5,7 @@
  * Then come the external dependencies:
  *
  *= require jquery.timepicker
+ *= require jquery-ui/theme
```

Comment:
I'm not sure if we can use @import to bring in external dependencies that are provided to us by gems. I switched us to using it for all of the scss files that we own, but using =require shouldn't be an issue for external stuff.